displays data from airline api now

gmoney3303 opened the pull request:
I added code to add the double search button as well as it now displays that information in the calendar on the day it is.

gmoney3303 committed:
displays data from airline api now

chilejay7 reviewed (changes requested):
Austin, please check your code against the main branch.  This PR is trying to update certain aspects of Aubrey's calendar code that will potentially cause a conflict when others pull from main if this is merged.

gmoney3303 committed:
Merge branch 'main' into garretts-changes

chilejay7 committed:
Update script.js

Updated the script to incorporate the main code from the calendar section to avoid conflicts.

chilejay7 reviewed (approved):
Thanks for looking at this, Garrett.
